dockers/piped-base/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM alpine:3.12.1
 
-ENV PIPED_TOOLS_DIR=/tools
+ENV PIPED_SHARED_HOME=/etc/pipecd
```

Comment:
Actually, this is not a shared home, just a shared directory.
So I prefer `PIPED_DIR=/etc/piped` to this one. (and `piped` instead of `pipecd`)

Reply:
Thank you. I was honestly wondering if this name is suitable. Let me fix it as you proposed.